resources/help/mud.md

```diff
@@ -126,3 +126,6 @@ mud.add_tag("MSDP")
 
 The primary use for this function is to indicate what telnet protocols are
 active for the current mud.
+
```

Comment:
We need a '##' between these two specs to conform to the "format".

Reply:
Oops, missed that.  Fixed